es_ES/cookbook/design/themes.md

```diff
@@ -0,0 +1,161 @@
+---
+layout: page
+title: "Uso de Themes para compartir colores y estilos de fuente"
+permalink: /cookbook/design/themes/
+---
+
+Para compartir colores y estilos de fuente a través de nuestra aplicación, podemos aprovechar los temas. Hay dos formas de definir temas: App-wide o usando los Widgets 
+`Theme` que definen los colores y estilos de fuente para una parte particular de nuestra aplicación. De hecho, los temas app-wide son simplemente Widgets `Theme` creados en la raíz de nuestras aplicaciones por `MaterialApp`! 
+
+Después de definir un Theme, podemos usarlo dentro de nuestros propios Widgets. Además, los 
+Material Widgets proporcionados por Flutter utilizarán nuestro Theme para establecer los colores de fondo y los estilos de fuente para AppBars, Buttons, Checkboxes, y más.    
+
+## Creando una app theme
+
+Para compartir un Theme que contenga colores y estilos de fuente en toda nuestra aplicación, podemos proporcionar [`ThemeData`](https://docs.flutter.io/flutter/material/ThemeData-class.html)
+al constructor de `MaterialApp` .
+
+Si no se proporciona ningún `theme` , Flutter creará un tema alternativo bajo la cubierta.
```

Comment:
a fallback theme under the hood.
un tema alternativo por nosotros. 
La traducción literal de la frase hecha en ingles suena extraña en español

Reply:
En realidad el autor lo que indica es que por defecto siempre se creara un theme, pero utilizó una expresión. De todas maneras con tu sugerencia se cubre lo que el autor quiere designar. Gracias